### PR TITLE
removed go-bindata from pkg/operator

### PR DESCRIPTION
**Pull request by facchettos**
### Which issue this PR addresses:
Removes go-bindata and replaces it with the standard std/embed 

Plan is to remove all occurence of gobindata in the repo

### CODE REVIEW

**Comment by m1kola** [comment]
Just yesterday I had a chat with @petrkotas about replacing go-bindata with standard library. I beleive he was going to create ticket for doing this across codebase. Pleae catch up with him.

File: pkg/util/embed/embeddedfiles.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+func ReadDirRecursive(fs embed.FS, dir string) [][]byte {
+	result := make([][]byte, 0)
+	entries, err := fs.ReadDir(dir)
+	if err != nil {
+		return nil
+	}
+	for _, v := range entries {
+		if v.IsDir() {
+			result = append(result, ReadDirRecursive(fs, dir+"/"+v.Name())...)
```
Comment (hawkowl) [inline pkg/util/embed/embeddedfiles.go:18]: we should use https://pkg.go.dev/path/filepath#Join for this and elsewhere, imo

**Review by s-amann — APPROVED** [review]
this will be a nice win once all bindata is gone!

**Review by SudoBrendan — APPROVED** [review]
This looks awesome! If you didn't plan to already, you should sign up for a demo slot to show this off.

**Review by s-amann — APPROVED** [review]

**Review by petrkotas — APPROVED** [review]
LGTM

**Pull request merged**